include/odp/api/spec/init.h

```diff
@@ -341,6 +346,39 @@ int odp_term_local(void);
  */
 int odp_term_global(odp_instance_t instance);
 
+/**
+ * Abnormal ODP termination after a non-recoverable error
+ *
+ * Application may use this function to terminate an ODP instance after facing
+ * a non-recoverable error. Depending on the implementation, this function may
+ * attempt to dump stack and other memory areas, clean up and stop HW
+ * operations and/or perform other actions helpful in postmortem analysis.
+ * Depending on the nature of the error resulting in the abnormal termination,
+ * these actions may partially or completely fail. Flags (ODP_TERM_*) and data
+ * parameters can be used to control and pass additional information to the
+ * termination process.
+ *
+ * Some coordination across threads is required when abnormally terminating, if
+ * other threads continue calling ODP functions during or after termination,
+ * their operation is most likely affected.
+ *
+ * When the function returns, the ODP instance has been destroyed either
+ * completely or partially. Application must not attempt to call any ODP
+ * functions during its remaining lifetime, but terminate as soon as feasible.
+ *
+ * @param instance        Instance handle
+ * @param flags           A bit mask of control flags (ODP_TERM_*), set to 0
+ *                        when no flags
+ * @param data            Additional data, set to 0 when no additional data
```

Comment:
In order to scale more, may be config structure can be passed(sub structure per FLAG). Just in case, uint64_t data is enough in the future. No strong opinion, though.

Reply:
If some of the flags are implementation-specific, defining a structure to pass here might be challenging. `uint64_t` does accommodate a pointer so in case something more is needed, it could be flag-specific pointer to some struct.

Reply:
OK